app/controllers/admin/deploy_groups_controller.rb

```diff
@@ -72,6 +73,22 @@ def deploy_all_now
     redirect_to deploys_path(ids: deploys.map(&:id))
   end
 
+  def create_all_stages
+    deploy_group.environment.template_stages.find_each do |template_stage|
```

Comment:
a 1-off way of doing things ... inconsistent with the old way ... will be a confusing UI ... 
reuse the old UI ... only modify the way it gets data ?